libcxx/test/std/input.output/file.streams/fstreams/filebuf.virtuals/setbuf.pass.cpp

```diff
@@ -0,0 +1,118 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <fstream>
+
```

Comment:
I think this requires `// UNSUPPORTED: no-filesystem` and maybe `no-localization`.

Reply:
Actually it seems we don't other tests in this directory do not have `UNSUPPORTED`. Obviously I let the CI be the final arbitrator.